python_modules/libraries/dagster-fivetran/dagster_fivetran_tests/test_assets.py

```diff
@@ -39,30 +39,41 @@ def test_fivetran_asset_run(asset_keys, should_error):
     final_data = {"succeeded_at": "2021-01-01T02:00:00.0Z"}
     api_prefix = f"{FIVETRAN_API_BASE}/{FIVETRAN_CONNECTOR_PATH}{DEFAULT_CONNECTOR_ID}"
 
+    fivetran_assets = build_fivetran_assets(
+        name="my_cool_ft_assets",
+        connector_id=DEFAULT_CONNECTOR_ID,
+        asset_keys=asset_keys,
+        poll_interval=0.1,
+        poll_timeout=10,
+    )
+
+    # expect the multi asset to have one asset key and one output for each specified asset key
+    assert fivetran_assets.asset_keys == set(asset_keys)
+    assert len(fivetran_assets.op.output_defs) == len(asset_keys)
```

Comment:
And I guess one more useful assertion would be that `schema1.table2` _doesn't_ exist as either an asset key or output def at this point (but that it will exist after we run the job). Might require bumping some of the test setup (response mocking) up a little higher to make it more clear why we expect that. And perhaps renaming the tables to make their relationship to the test a bit more clear (`schema.tracked_table`, `schema.untracked_table`, etc.)